File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/EurSellComparator.java
package fobo66.exchangecourcesbelarus.util.comparators;

import fobo66.exchangecourcesbelarus.models.Currency;

public class EurSellComparator implements CurrencyComparator {
  @Override public int compare(Currency lhs, Currency rhs) {
    return rhs.eurSell.compareToIgnoreCase(lhs.eurSell);
  }
}

File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/EurSellComparator.kt
package fobo66.exchangecourcesbelarus.util.comparators

import fobo66.exchangecourcesbelarus.entities.Currency

class EurSellComparator : Comparator<Currency> {
  override fun compare(lhs: Currency, rhs: Currency): Int {
    return rhs.eurSell.compareTo(lhs.eurSell, ignoreCase = true)
  }
}
